packages/load-script/src/dom-operations.js

```diff
@@ -3,14 +3,26 @@ import { handleRequestError, handleRequestSuccess } from './callback-handler';
 
 const debug = debugFactory( 'lib/load-script/dom-operations' );
 
-export function createScriptElement( url ) {
+export function createScriptElement( url, args ) {
 	debug( `Creating script element for "${ url }"` );
 	const script = document.createElement( 'script' );
+
 	script.src = url;
 	script.type = 'text/javascript';
 	script.async = true;
 	script.onload = handleRequestSuccess;
 	script.onerror = handleRequestError;
```

Comment:
One thing we could do is set those after `args` if `args` haven't set them already. Maybe that will help pass the test failure you're having.

Reply:
I did that specifically for 'async' because otherwise it gets quirky fast... I can't check for the existence those props `in script` because `script` will have them (or at least some) set by default.